rebasehelper/helpers/copr_helper.py

```diff
@@ -54,18 +54,27 @@ def get_client(cls):
             return client
 
     @classmethod
-    def create_project(cls, client, project, chroot, description, instructions, permanent=False, hide=True):
+    def create_project(cls, client, project, chroots, description, instructions, permanent=False, hide=True):
         try:
             client.project_proxy.add(ownername=client.config.get('username'),
                                      projectname=project,
-                                     chroots=[chroot],
+                                     chroots=chroots,
                                      delete_after_days=None if permanent else cls.DELETE_PROJECT_AFTER,
                                      unlisted_on_hp=hide,
                                      description=description,
                                      instructions=instructions)
-        except CoprRequestException:
-            # reuse existing project
-            pass
+        except CoprRequestException as e:
+            error = e.result.error
+            try:
+                error = next(iter(error.values()))[0]
```

Comment:
I'm still suspicious about this. How does the error *object* look like?

Reply:
Ok, so it really is a dict containing single-item list :roll_eyes: 
In that case I think this is much more readable:
```python
[[error]] = error.values()
```